README.md

```diff
@@ -1,227 +1,42 @@
-   ```
-
-   Approximate output:
-   ```
-   usage: ote eval [-h] --test-ann-files TEST_ANN_FILES --test-data-roots TEST_DATA_ROOTS --load-weights LOAD_WEIGHTS template {params} ...
-
-   positional arguments:
-     template
-     {params}              sub-command help
-      params              Hyper parameters defined in template file.
-
-   optional arguments:
-     -h, --help            show this help message and exit
-     --test-ann-files TEST_ANN_FILES
-                           Comma-separated paths to test annotation files.
-     --test-data-roots TEST_DATA_ROOTS
-                           Comma-separated paths to test data folders.
-     --load-weights LOAD_WEIGHTS
-                           Load only weights from previously saved checkpoint
-   ```
-
-5. Let's have a look at `ote export` help. These parameters are the same for all model templates.
-   ```
-   ote export ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --help
-   ```
-
-   Approximate output:
-   ```
-   usage: ote export [-h] --load-weights LOAD_WEIGHTS --save-model-to SAVE_MODEL_TO [--ann-files ANN_FILES] [--labels LABELS [LABELS ...]] template
-
-   positional arguments:
-     template
+## Repository components
+* [OTE SDK](ote_sdk)
+* [OTE CLI](ote_cli)
+* [OTE Algorithms](external)
 
-   optional arguments:
-     -h, --help            show this help message and exit
-     --load-weights LOAD_WEIGHTS
-                           Load only weights from previously saved checkpoint
-     --save-model-to SAVE_MODEL_TO
-                           Location where exported model will be stored.
-     --ann-files ANN_FILES
-     --labels LABELS [LABELS ...]
-   ```
+## Quick start guide
+In order to get started with OpenVINO™ Training Extensions click [here](QUICK_START_GUIDE.md).
 
-### OTE Jupyter Nootebooks
-One can use Jupyter notebooks or OTE CLI tools to start working with models:
-```
-pip3 install notebook; cd ote_cli/notebooks/; jupyter notebook
-```
+## License
+Deep Learning Deployment Toolkit is licensed under [Apache License Version 2.0](LICENSE).
+By contributing to the project, you agree to the license and copyright terms therein
+and release your contribution under these terms.
 
 ## Misc
 
-Models that were previously developed can be found [here](misc).
+Models that were previously developed can be found [here](https://github.com/openvinotoolkit/training_extensions/tree/misc).
 
 ## Contributing
 
-Please read the [contribution guidelines](CONTRIBUTING.md) before starting work on a pull request.
+Please read the [Contributing guide](CONTRIBUTING.md) before starting work on a pull request.
 
-## Known Limitations
+## Known limitations
 
 Currently, training, exporting, evaluation scripts for TensorFlow\*-based models and the most of PyTorch\*-based models from [Misc](#misc) section are exploratory and are not validated.
```

Comment:
Sorry, but `misc` is not a section any more. It is a separate branch with old stuff.  
Unfortunately, it seems to me that this phrase should be re-formulated.

Reply:
fixed